app/src/main/java/com/ucmobiledevelopment/freeways/MainActivity.kt

```diff
@@ -163,7 +163,7 @@ class MainActivity : ComponentActivity() {
                 )
                 Button(
                     onClick = {
-                        var incidentInfo = Incident().apply {
+                        val incidentInfo = Incident().apply {
 
                             stateId =  0
                             stateName = inStateName
```

Comment:
Data within incidentInfo is seems immutable, so I changed it to a val instead of a var.

Reply:
Good idea.  Default to val.  Only use var if you know you need to.